### PR TITLE
Register fixes

### PR DESCRIPTION
**Pull request by ilzeLourens**

### CODE REVIEW

File: skietbaan_fe/src/components/Login.js [hunk 1]
```diff
@@ -158,7 +169,7 @@ class Login extends Component {
   }
 
   goToRegister() {
-    window.location = "/Register-page"
+    window.location = "/RegisterPage"
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/Login.js]: camelCase please not CapitalCase

Comment (mgaud) [reply skietbaan_fe/src/components/Login.js]: why
its a url, it should all be lowercase

Comment (mgaud) [reply skietbaan_fe/src/components/Login.js]: https://wiredimpact.com/blog/never-use-capital-letters-urls/

Comment (ilzeLourens) [reply skietbaan_fe/src/components/Login.js]: you guys

File: skietbaan_fe/src/components/RegisterStyles.css [hunk 2]
```diff
@@ -178,25 +191,34 @@
 .welcome-header{
   width: 100%;
   height: 79px;
-  background-color: #EA241A;
+  background-color: #6F839E;
   text-align: center;
+  border-bottom-right-radius: 20px;
+  border-bottom-left-radius: 20px;
+  box-shadow: 0px 10px 16px -6px rgba(0,0,0,0.33);
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/RegisterStyles.css]: use hex instead and empty life EOF

**Pull request merged**